pkg/core/transaction/transaction.go

```diff
@@ -118,6 +121,16 @@ func (t *Transaction) DecodeBinary(br *io.BinReader) {
 	t.Type = TXType(br.ReadB())
 	t.Version = uint8(br.ReadB())
 	t.Nonce = br.ReadU32LE()
+
+	bytes := make([]byte, util.Uint160Size)
+	br.ReadBytes(bytes)
+	sender, err := util.Uint160DecodeBytesLE(bytes)
```

Comment:
1) It should be BE
2) Take a look at `output.go` and the way `ScriptHash` is being handled there, way simpler than this code.